lib/teach-api.js

```diff
@@ -73,6 +73,20 @@ _.extend(TeachAPI.prototype, {
     }
 
     return req;
+  },
+  getAllData: function(callback) {
+    request('get', process.env.TEACH_API_URL || this.baseURL + "/api/clubs")
```

Comment:
Oh, you should just use `this.baseURL` here, as its value already takes `process.env.TEACH_API_URL` into account.